### ISSUE TITLE
Cortical microinfarct

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7045]
**Preferred term label:**

Cortical microinfarct

**Synonyms**
Cerebral cortical microinfarct

**Definition (free text, please give PubMed ID)**

An infarct located on the outer layer of the cerebrum that can only be observed microscopically.
PMID:10867785

**Parent term (use hpo.jax.org/app)**

abnormal cerebral cortex morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Alzheimer's Disease (OMIM: 104300)

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
@sarahstofel  How is this feature related a Lacunar stroke? Both are "small"
https://hpo.jax.org/app/browse/term/HP:0032325

**Comment by pnrobinson** [comment]
@sarahstofel thanks, adding with expanded definition

An infarct located on the outer layer of the cerebrum that can only be observed microscopically. Cerebral microinfarcts are typically defined as sharply delimited microscopic regions of cellular death or tissue necrosis, sometimes with cavitation (that is, a central fluid-filled cavity). The term “microscopic” denotes that these lesions are not visible by gross inspection of the brain but seen by light microscopy. The term infarct is most commonly used for ischemia-related tissue loss, and indeed the pathologic appearance of microinfarcts is consistent with that of known ischemic infarctions.
(see https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3359329/)
